consensus/service/src/api/blockchain_api_service.rs

```diff
@@ -62,7 +64,16 @@ impl<L: Ledger + Clone> BlockchainApiService<L> {
         let num_blocks = self.ledger.num_blocks()?;
         let mut resp = LastBlockInfoResponse::new();
         resp.set_index(num_blocks - 1);
-        resp.set_minimum_fee(self.minimum_fee);
+        resp.set_mob_minimum_fee(
+            self.fee_map
+                .get_fee_for_token(&Mob::ID)
+                .expect("should always have a fee for MOB"),
```

Comment:
It seems better to return an Error here rather than panicking.

Reply:
I'm not sure about that since that will require a new error type (can't use mc_ledger_db::Error for this), and the `FeeMap` does enforce the existence of a minimum fee for mob.
If you don't like that, what about changing it to default to zero if this doesn't exist + logging an error?

Reply:
Reviewing `FeeMap`, I agree it enforces that requirement, so the panic seems unreachable.